scripts/gef-extras.sh

```diff
@@ -1,17 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
 if [ $# -ge 1 ]; then
-  DIR="$(realpath $1)/gef-extras"
-  test -d ${DIR} || exit 1
-elif [ -d ${HOME}/.config ]; then
+  DIR="$(realpath "$1")/gef-extras"
```

Comment:
`realpath` isn't installed by default on OSX so maybe we want to use something else.

Reply:
I have no experience with OS X and can't test there so what would "something else" be in this case?

Reply:
quick web search led me to [this post on stackoverflow](https://stackoverflow.com/questions/3572030/bash-script-absolute-path-with-os-x/3572105#3572105). 

Would this be better?

Reply:
Let's wait until later, we don't have good osx support anyway.